pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -391,7 +393,7 @@ func (o *EditOptions) visitToApplyEditPatch(originalInfos []*resource.Info, patc
 			}
 		}
 		if originalInfo == nil {
-			return fmt.Errorf("no original object found for %#v", info.Object)
+			return fmt.Errorf("no original object found")
```

Comment:
See [this bazel test](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/50205/pull-kubernetes-bazel/34592/).

This test passed on Mac. But failed due to this disordered string. Kind of weird that `%#v` produces different Go-syntax representation on Mac and Linux.